xDrip/Scenes/Settings/BaseSettings/BaseSettingsViewController.swift

```diff
@@ -10,21 +10,35 @@ import UIKit
 
 class BaseSettingsViewController: UIViewController {
     private lazy var tableView: UITableView = {
-        let tableView = UITableView(frame: .zero, style: .grouped)
+        var tableView: UITableView
+        
+        if #available(iOS 13.0, *) {
+            tableView = UITableView(frame: .zero, style: isRootSettings && UIDevice.current.userInterfaceIdiom == .pad ? .grouped :.insetGrouped)
+        } else {
+            tableView = UITableView(frame: .zero, style: .grouped)
+        }
+        
         tableView.translatesAutoresizingMaskIntoConstraints = false
         tableView.delegate = self
         tableView.dataSource = self
         view.addSubview(tableView)
         tableView.bindToSuperview()
         
         tableView.registerNib(type: BaseSettingsDisclosureCell.self)
+        tableView.registerNib(type: BaseSettingsSingleSelectionTableViewCell.self)
+        tableView.registerNib(type: BaseSettingsRightSwitchTableViewCell.self)
+        tableView.registerNib(type: BaseSettingsVolumeSliderTableViewCell.self)
+        tableView.registerNib(type: BaseSettingsPickerExpandableTableViewCell.self)
+        tableView.registerNib(type: BaseSettingsTextInputTableViewCell.self)
         
         return tableView
     }()
     
     private var viewModel: BaseSettings.ViewModel?
     private let cellFactory = BaseSettingsCellFactory()
     
+    var isRootSettings = false
```

Comment:
Instead of creating this variable, let's define table view style explicitly. Create a readonly variable in `BaseSettingsViewController` - `tableViewStyle: UITableView.Style` (or whatever type is). Set it to `.group` by default. Override it in root settings.